### ISSUE TITLE
Extract doc-strings from members of a class PVGroup

### ISSUE THREAD

**Issue by vstadnytskyi** [caproto/caproto#558]
I am writing documentation for some of my caproto test/example servers. I have used the following command to extract all docstrings from a python Class.

```
.. autoclass:: caproto_sandbox.example_1_server.Choices
  :members:
```

This doesn't with `class Server(PVGroup)` It doesn't show any members. It would be nice to have documentation assembled directly from docstrings. See my .rst file https://github.com/vstadnytskyi/caproto-sandbox/blob/master/docs/source/example_1.rst

and an example of documentation. https://vstadnytskyi.github.io/caproto-sandbox/example_1.html

Is there a way to programmatically extract docstrings?

**Comment by danielballan** [comment]
Does using `:undoc-members:` make any difference?

**Comment by vstadnytskyi** [comment]
I tried 

```
.. automodule:: caproto_sandbox.example_1_server
  :members:
  :undoc-members:
  :show-inheritance:
```

and I get 

<img width="769" alt="image" src="https://user-images.githubusercontent.com/29989124/73677557-2478b180-467c-11ea-84d8-5b13d9b69673.png">

or see the github link [this](https://vstadnytskyi.github.io/caproto-sandbox/example_1.html#caproto_sandbox.example_1_server.Server), where example_1_server.Server is

```
class Server(PVGroup):
    io_put_queue = None

    running = pvproperty(value=1)
    jog = pvproperty(value=0.0)
    enum_strings=['Insert','Retract',' ']
    choices = pvproperty(value=' ',enum_strings=enum_strings , dtype=ChannelType.ENUM)

...
```

Is it possible it actually gets docstring from PVGroup?

**Comment by klauer** [comment]
This should be resolved in v0.6.0. Instead of returning the confusing `PVSpec`, accessing the property will return the `pvproperty` instance itself.
